README.md

```diff
@@ -94,7 +94,7 @@ specific language governing permissions and limitations under the License.
 
 <!-- refs -->
 [github-actions]: https://github.com/linkerd/linkerd2/actions
-[github-actions-badge]: https://github.com/linkerd/linkerd2/workflows/CI/badge.svg
+[github-actions-badge]: https://github.com/linkerd/linkerd2/workflows/Release/badge.svg
```

Comment:
How about referencing Cloud Integration workflow? That runs on all merges to main.

Release will only run on commits that are tagged. There is a possibility that an untagged commit breaks Cloud Integration, but it would still show as succeeding because it is looking at some previous Release workflow.

Reply:
`Cloud Integration` seems like a more appropriate workflow to show a badge for. Part of the reason for selecting `Release` was because the name of the workflow appears in the badge. Just confirming we're ok with a badge that says `Cloud Integration`:
<img width="1249" alt="Screen Shot 2020-07-20 at 4 48 51 PM" src="https://user-images.githubusercontent.com/236915/87997069-ec89ed00-caa8-11ea-9c18-b7c8333921b0.png">

Reply:
Ah yep that is a good thing to note. I'm okay with that though. I think having a `Cloud Integration` label that is reflective of the current commit in main is better than `Release` that may reflect something older. Thanks for pointing that out!